docs/source/documentation.rst

```diff
@@ -536,56 +513,6 @@ Here is a list of useful Jupyter notebooks available with AxonDeepSeg:
 
    **Option:** If your image to segment is too small, you can use padding to artificially increase its size (i.e. add empty pixels around the borders).
 
-Guide for manual labelling
```

Comment:
Oh! we will need a plan for this part.

So, now that we do the training in ivadomed:
- shouldn't we mention this somewhere and link to the tutorial?
- people will still need a "guide" for manual labelling, so the information here should be adjusted to the need of ivadomed for training, the question is, here or on ivadomed side?
- the differences are:
    - ivadomed use 2 separate black and white masks
    - ivaodmed uses BIDS structure (I'm a little bit stuck for documenting this part at the moment because the BEP is not merged yet).

I'll think about it!

Reply:
I deleted it, because I assumed AxonDeepSeg doesn't deal with training at all anymore. I could see a simple link to an IVADOMED tutorial, but am unsure if we should include it in our own documentation as it might get out of sync quickly with IVADOMED.

Your call!

Reply:
Maybe @jcohenadad can chime in too with his opinion on this